flixel/tweens/FlxTween.hx

```diff
@@ -723,6 +799,28 @@ class FlxTweenManager extends FlxBasic
 		
 		return Tween;
 	}
+	
+	/**
+	 * Remove a FlxTween without destroying it
+	 * 
+	 * @param	Tween		The FlxTween to remove.
+	 * @param	Destroy		Whether you want to destroy the FlxTween.
+	 * @return	The added FlxTween object.
+	 */
+	@:allow(flixel.tweens.FlxTween)
+	private function removeSoft(Tween:FlxTween):FlxTween
```

Comment:
Might make sense to bring back the Destroy flag to the original `remove()` considering this is a 1:1 copy of that otherwise.